### ISSUE TITLE
More pathing (flooring?) Decorative stones?

### ISSUE THREAD

**Issue by Katknapp** [APICO-Modders/APICO-Community-Decorations#10]
I have a really forest nature flavor to my game and I can't help but think, it would be so cute if there were more path options. I have paths through my forest (see screenshot below). When I walk them, I can't help but imagine some cute cobble stone, or dirt/natural flooring to line accent the paths. Maybe even stones to decorate with since you lose the large boulder if you mine them. Maybe even grass. I hope that makes sense.
<img width="627" alt="Screen Shot 2022-05-23 at 8 39 50 PM" src="https://user-images.githubusercontent.com/14977930/169932033-0ad89750-8210-44c8-bdbe-32b93c16746f.png">

**Comment by ThePebblemancer** [comment]
I've been told that flooring is impossible to make rn :(

**Comment by Katknapp** [comment]
> I've been told that flooring is impossible to make rn :(

Oh that's a shame! It's early days, so we'll see later. ;) Thanks again for checking at least!

**Comment by Vildravn** [comment]
Yeah, new tiles like walls and floors are not really doable at least right now. That said, we might be able to fake something in as I have some ideas :)
